docs/cmdline-opts/etag-compare.d

```diff
@@ -0,0 +1,17 @@
+Long: etag-compare
+Arg: <file>
+Help: Pass an ETag from a file as a custom header
+Protocols: HTTP
+---
+This option can be used to read an ETag from a file, and make
```

Comment:
can be used to...?  It *is* used for it and not for anything else. How about a more direct:

"This option makes a conditional HTTP request for the specific ETag read from the given file" ?